website/content/docs/security/acl/tokens/create/create-a-dns-token.mdx

```diff
@@ -0,0 +1,329 @@
+---
+layout: docs
+page_title: Create tokens for service registration
+description: >-
+  Learn how to create ACL tokens to enable Consul DNS.
+---
+
+# Create a DNS token
+
+This topic describes how to create a token that you can use to enable using Consul DNS.
+
+## Introduction
+
+A Consul agent must be configured with a token linked to policies that grant the appropriate set of permissions. To enable catalog lookups over DNS, the token must be linked to policies that grant the following permissions:
+
+Specify the [`default`](/consul/docs/agent/config/config-files#acl_tokens_default) token to the Consul agent to authorize the agent to respond to DNS queries. Refer to [DNS usage overview](/consul/docs/services/discovery/dns-overview) for details on configuring and using Consul DNS.
```

Comment:
```suggestion

A Consul agent must be configured with a token linked to policies that grant the appropriate set of permissions. 

Specify the [`default`](/consul/docs/agent/config/config-files#acl_tokens_default) token to the Consul agent to authorize the agent to respond to DNS queries. Refer to [DNS usage overview](/consul/docs/services/discovery/dns-overview) for details on configuring and using Consul DNS.
```

I'm not sure if this is the right intro content for this topic. I think we're missing a statement that connects the concept of the agent to the DNS. Maybe something like: 

"The Consul binary ships with a DNS server that you can use for service discovery in your network. The agent that fulfills DNS lookups requires appropriate ACL permissions to services registered in Consul."  

I don't know if that's exactly how it works, but I think the __why__ is missing.

Reply:
👍 I like the suggestion. I added it and slightly modified it to the following (modification in italics):

> The Consul binary ships with a DNS server that you can use for service discovery in your network. The agent that fulfills DNS lookups requires appropriate ACL permissions to _discover services, nodes, and prepared queries_ registered in Consul.